cmd/storage_test.go

```diff
@@ -92,3 +96,22 @@ func Test_StorageCmdOutput(t *testing.T) {
 		assert.Equal(t, test.expectedOutput, string(out))
 	}
 }
+
+func Test_StorageCmdDeleteOutput(t *testing.T) {
+	type testCase struct {
+		expectedOutput string
+	}
+	err := testCase{expectedOutput: ""}
+	r, w, _ := os.Pipe()
+	// Change the standard output to the write side of the pipe
+	os.Stdout = w
+	mockClient := mockStorageGetter{}
+	cmd := produceStorageCmdRunFunc(mockClient, storageDeleteAction)
+	cmd(new(cobra.Command), []string{"rm", mockStorage.Properties.ObjectUUID})
+	// close the write side of the pipe so that we can start reading from
+	// the read side of the pipe
+	w.Close()
+	// Read the standard output's result from the read side of the pipe
+	out, _ := ioutil.ReadAll(r)
+	assert.Equal(t, err.expectedOutput, string(out))
```

Comment:
Can you remove the source comments here? ode is pretty self-explanatory. Only here please.

Reply:
As far I understood, only removed the following comment: `// Read the standard output's result from the read side of the pipe` ?